pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -6175,7 +6173,8 @@ func TestChangefeedJobUpdateFailsIfNotClaimed(t *testing.T) {
 		// another node.
 		sqlDB.Exec(t, `UPDATE system.jobs SET claim_session_id = NULL WHERE id = $1`, jobID)
 
-		timeout := 5 * time.Second
+		timeout := (5 * time.Second) + changefeedbase.Quantize.Get(&s.Server.ClusterSettings().SV)
+
 		if util.RaceEnabled {
 			timeout = 30 * time.Second
```

Comment:
Do we need to add the quantization factor here too?

Reply:
Done.